specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -0,0 +1,5553 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "securityPolicyParameters",
+            "in": "body",
+            "description": "Security policy update properties",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SecurityPolicyParameters"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityPolicy"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/SecurityPolicy"
+            }
+          },
+          "default": {
+            "description": "CDN error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./cdn.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "SecurityPolicies"
+        ],
+        "description": "Deletes an existing security policy within profile.",
+        "operationId": "SecurityPolicies_Delete",
+        "x-ms-examples": {
+          "SecurityPolicies_Delete": {
+            "$ref": "./examples/SecurityPolicies_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "description": "Name of the CDN profile which is unique within the resource group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "securityPolicyName",
+            "in": "path",
+            "description": "Name of the Secret under the profile.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "CDN error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./cdn.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/secrets": {
```

Comment:
Does it return any secret (e.g. password, key, token, connection string, etc)

Reply:
Nope. It will be a reference to the key vault url.